src/Components/App/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
 import './App.css';
+// import LoginForm from '../LoginForm/LoginForm';
+import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+import LoginButton from '../LoginButton/LoginButton';
+import LogoutButton from '../LogoutButton/LogoutButton';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      {/* <LoginForm /> */}
```

Comment:
You would have saved time if you would have read the documentation first instead of guessing how the application would run.  I would continue reading through the documentation before you start adding your own components or CSS.